2-mandatory/1-weather-report.js

```diff
@@ -13,9 +13,25 @@
 
 function getTemperatureReport(cities) {
     // TODO
-}
-
+    let arr = [];
+  let degree = 0;
+  let cityDegree = "";
+  for (let i = 0; i < cities.length; i++) {
+    degree = temperatureService(cities[i]);
+    
+    arr.push(cityDegree);
+  }
+  return arr;
 
+} let arr = [];
```

Comment:
It looks like you have the same code copied twice here, maybe this was an accident?